src/site/content/en/blog/gamepad/index.md

```diff
@@ -0,0 +1,347 @@
+  */
+});
+```
+
+### Being notified when a gamepad gets disconnected
+
+Being notified of gamepad disconnects happens analogously to the way connections are detected.
+Just this time the app listens for the `gamepaddisconnect` event.
+Note how in the example below `connected` is now `false`
+when I unplug the Xbox 360 controller from the previous sample.
+
+```js/6
+window.addEventListener('gamepadconnected', (event) => {
+  console.log('❌ 🎮 A gamepad was disconnected:', event.gamepad);
+  /*
+    gamepad: Gamepad
+    axes: (4) [0, 0, 0, 0]
+    buttons: (17) [GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton, GamepadButton]
+    connected: false
+    id: "Xbox 360 Controller (STANDARD GAMEPAD Vendor: 045e Product: 028e)"
+    index: 0
+    mapping: "standard"
+    timestamp: 6563054.284999998
+    vibrationActuator: null
+  */
+});
+```
+
+### The gamepad in your game loop
+
+Getting a hold of a gamepad starts with a call to `navigator.getGamepads()`,
+which returns a `GamepadList` object with `Gamepad` items.
+The `GamepadList` object in Chrome *always* has a fixed length of four items.
+If zero or less than four gamepads are connected, an item may just be `null`.
+Always be sure to check all items of the `GamepadList` and be aware that
+gamepads "remember" their slot and may not always be present at the first available slot.
+
+```js
+// When no gamepads are connected:
+navigator.getGamepads()
+// GamepadList {0: null, 1: null, 2: null, 3: null, length: 4}
+```
+
+If one or several gamepads are connected,
+but `navigator.getGamepads()` still reports `null` items,
+you may need to "wake" each gamepad by pressing any of its buttons.
+You can then poll the gamepad states in your game loop as shown below.
+
+```js
+const pollGamepad = () => {
+  // Always call `navigator.getGamepads()` inside of
+  // the game loop, not outside.
+  const gamepads = navigator.getGamepads();
+  for (const gamepad of gamepads) {
+    // Disregard empty slots.
+    if (!gamepad) {
+      continue;
+    }
+    // Process the gamepad state.
+    console.log(gamepad);
+  }
+  // Call yourself upon the next animation frame.
+  // (Typically this happens every 60 times per second.)
+  window.requestAnimationFrame(pollGamepad)
+};
+// Kick off the initial game loop iteration.
+pollGamepad();
+```
+
+{% Aside 'gotchas' %}
+  Do not store a lasting reference to the `GamepadList` result *outside* of the game loop,
+  since the method returns a static snapshot, not a live object.
+  Call `navigator.getGamepads()` each time anew in your game loop.
+{% endAside %}
+
+### Making use of the vibration actuator
+
+The `vibrationActuator` field corresponds to a configuration of motors or other actuators
+that can apply a force for the purposes of haptic feedback.
```

Comment:
```suggestion
The `vibrationActuator` property returns a `GamepadHapticActuator`
object, which corresponds to a configuration of motors or other actuators
that can apply a force for the purposes of haptic feedback.
```

Reply:
Great job digging this up. This is the way it happens in Canary, not yet Beta. But for future-proofness I think it's fine to say how it will work. Thank you.